### ISSUE TITLE
Cannot show remote video

### ISSUE THREAD

**Issue by abdymm** [bluejeans/sdk-webrtc-meetings#15]
Hi @glenninn , i just try this example, and i cannot see the remote videos?
is that anything i should modify?

**Comment by dougalg** [comment]
Same problem here, but I was thinking it might be an issue with CORS. I used a plugin that allowed CORS, but I noticed that the websocket connections were still failing due to CORS errors, so I wonder if that is the problem.

It would be really nice to have a simple working out of the box implementation for testing...

**Comment by abdymm** [comment]
Yea yea i did use CORS extention, it successfully when try to connect the socket, but cannot receive the remote stream video i think

**Comment by vintageanand** [comment]
@abdymm @dougalg 

Thank you for raising this issue.

After investigation, we believe the issue with not receiving remote video in the sample app is tied to an underlying platform change rolled out to a subset of accounts recently (Meetings belonging to free trials or Basic plans are particularly impacted).

We are targeting an update to the WebRTC SDK (and corresponding test page) for next week and will update this thread once that is complete. Thank you for your patience and help in advance in confirming that our fix addresses the issue.

**Comment by dougalg** [comment]
@vintageanand thanks a lot for the feedback! Looking forward to the update.

**Comment by abdymm** [comment]
@vintageanand really appreciate it! thanks for the feedback! Looking forward to the update.

**Comment by vintageanand** [comment]
@abdymm @dougalg Please update to the latest SDK version (1.4) that was pushed a few minutes back (or try it out on the test page at https://github.com/bluejeans/sdk-webrtc-meetings). This should ensure 2 way video is flowing again for all your meetings and avoid the gray screen issue. I will leave this issue open till one of you confirm.

**Comment by naveenprakash74** [comment]
> @abdymm @dougalg Please update to the latest SDK version (1.4) that was pushed a few minutes back (or try it out on the test page at https://github.com/bluejeans/sdk-webrtc-meetings). This should ensure 2 way video is flowing again for all your meetings and avoid the gray screen issue. I will leave this issue open till one of you confirm.

thanks, its work for me!😀

**Comment by abdymm** [comment]
It works now, @vintageanand  thanks!

**Comment by abdymm** [comment]
hi sir @vintageanand , sorry it works on your demo page, but when i put in my own server, it doesn't work
<img width="1440" alt="Screen Shot 2020-08-03 at 09 19 25" src="https://user-images.githubusercontent.com/1695466/89137422-a170df00-d56a-11ea-806c-b3751ad39b02.png">
again its because the CORS block, you have any idea to solve this? thankyou

**Comment by glenninn** [comment]
Abdy,  The WebRTC code checks for CORS whitelisting.  If you are running this code, then theserver where you host it must be whitelisted with BlueJeans.  Alternatively for development,you can get browser extensions that turn off CORS.
-Glenn

    On Sunday, August 2, 2020, 6:21:50 PM PDT, Abdy Malik Mulky <notifications@github.com> wrote:  
 
 


hi sir @vintageanand , sorry it works on your demo page, but when i put in my own server, it doesn't work

again its because the CORS block, you have any idea to solve this? thankyou

**Comment by abdymm** [comment]
ahh i see, allright then let me try on our server that whitelisted with bluejeans, thanks! @glenninn
